cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -78,6 +78,10 @@ spec:
     # If you have multiple rook-ceph clusters in the same k8s cluster, choose the same namespace (ideally, namespace with prometheus
     # deployed) to set rulesNamespace for all the clusters. Otherwise, you will get duplicate alerts with multiple alert definitions.
     rulesNamespace: rook-ceph
+    # labels is a list of key-value pairs. It is injected into all the monitoring resources created by operator.
+    # These labels can be passed as LabelSelector to Prometheus
+    # labels:
```

Comment:
Instead of adding this new `monitoring.labels`, I'd suggest we use the existing [labels section](https://github.com/rook/rook/blob/release-1.5/cluster/examples/kubernetes/ceph/cluster.yaml#L161) under the root of the cephcluster. This means the admin would specify `labels.monitoring`.

Reply:
I didn't know we had such spec already. Added "monitoring" to the section.